platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/turtle/TurtleNetwork.java

```diff
@@ -83,8 +83,9 @@ public TurtleNetwork(
     @Override
     @NonNull
     public List<Node> addNodes(final int count) {
-        if (state != State.INIT) {
-            throw new IllegalStateException("Cannot add nodes after the network has been started.");
+        if (state != State.INIT && state != State.SHUTDOWN) {
```

Comment:
This does not seem to be right. We do not want to add nodes after the network has been shut down.

Reply:
Removed. It was a leftover from the previously suggested approach of restarting the turtle network. Now that we would just destroy it - it is not needed indeed.